cmd/ocsp-updater/main.go

```diff
@@ -168,27 +167,54 @@ func getQuestionsForShardList(count int) string {
 }
 
 func (updater *OCSPUpdater) findStaleOCSPResponses(oldestLastUpdatedTime time.Time, batchSize int) ([]core.CertificateStatus, error) {
-	params := make([]interface{}, 0)
-	params = append(params, oldestLastUpdatedTime)
+	args := make([]interface{}, 0)
+	args = append(args, oldestLastUpdatedTime)
 
-	// If serialSuffixes is unset, this will be deliberately a no-op
+	// If serialSuffixes is unset, this will be deliberately a no-op.
 	for _, c := range updater.serialSuffixes {
-		params = append(params, c)
+		args = append(args, c)
 	}
-	params = append(params, batchSize)
-
-	statuses, err := sa.SelectCertificateStatusMetadata(
-		updater.readOnlyDbMap,
-		updater.queryBody,
-		params...,
+	args = append(args, batchSize)
+
+	rows, err := updater.readOnlyDb.Query(
+		fmt.Sprintf(
+			"SELECT %s FROM certificateStatus %s", strings.Join(
+				sa.CertStatusMetadataFields(), ","), updater.queryBody,
```

Comment:
indentation is kinda wonky here; let's put the string and each of the format args on separate lines.
```suggestion
			"SELECT %s FROM certificateStatus %s",
			strings.Join(sa.CertStatusMetadataFields(), ","),
			updater.queryBody,
```

Reply:
Agreed!